apis/tests/chatglm_test.py

```diff
@@ -0,0 +1,13 @@
+import sys
+sys.path.append('/home/jerry_kon/my_chatglm/ChatLLM')
+from apis.api import ChatLLM
+chat = ChatLLM(
```

Comment:
for unit test, this is not available because we have no model in this project. I'm still figuring out the proper approach to solve this, so currently, just remove this test.